app/models/user.rb

```diff
@@ -1,11 +1,20 @@
+# app/models/user.rb
+
 class User < ApplicationRecord
+  # Devise modules including :confirmable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable, :confirmable
```

Comment:
- [ ] Great job you added confirmable as mentioned but still the confirmable functionality is not working as expected since after signing up it leads me to user index page without asking for confirmation my email, While everything seems to be setup correctly and email confirmation is done as expected, you seem to be missing this line  in Application controller file `before_action :authenticate_user!` added here which will require you to first confirm your email before doing anything in the application. 🚀🎉✨🔥💯



https://github.com/mumbereh/Blog_App/assets/58553711/c58fa30c-da96-4c22-ae2f-06740d7cc738